telcell/data/models.py

```diff
@@ -44,6 +45,29 @@ def __iter__(self) -> Iterator[Measurement]:
         return iter(self.measurements)
 
 
+class MeasurementPair:
```

Comment:
Oude comment van Jeroen:
Kan deze hele class uberhaupt niet weg en dat we gewoon Tuple[Measurement, Measurement] gebruiken voor een measurement pair. Dat scheelt weer een class waar mensen over na hoeven te denken.

Het enige mogelijke nadeel dat ik zie is dat je van twee Measurements niet kunt bepalen of ze colocated zijn zonder dat je de bijbehorende Tracks hebt. Dat maakt het misschien voor nu nog een beetje onhandig...

Reply:
Ik vond het wel handig dat het een class was. Want dan kun je de eigenschappen zoals de time difference en colocation daar aan hangen en was het niet een tuple die deze eigenschappen wel of niet heeft. Wat doen we hiermee?